files/zh-cn/learn/performance/html/index.md

```diff
@@ -0,0 +1,270 @@
+
+在下面的章节中，我们将总结如何实现响应式的替代元素。你可以在[视频和音频内容](/zh-CN/docs/Learn/HTML/Multimedia_and_embedding/Video_and_audio_content)和[响应式图像](/zh-CN/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images)指南中找到更多关于这些实现的详细信息。
+
+### 通过 srcset 提供不同的图像分辨率
+
+为了根据设备的分辨率和视口大小提供相同图像的不同分辨率版本，你可以利用 [`srcset`](/zh-CN/docs/Web/HTML/Element/img#srcset) 和 [`sizes`](/zh-CN/docs/Web/HTML/Element/img#sizes) 这两个属性。
+
+以下示例为不同屏幕宽度提供了不同尺寸的图像：
+
+```html
+<img
+  srcset="480w.jpg 480w, 800w.jpg 800w"
+  sizes="(max-width: 600px) 480px,
+         800px"
+  src="800w.jpg"
+  alt="家庭肖像" />
+```
+
+`srcset` 提供源图像的固有尺寸以及它们的文件名，而 `sizes` 在每种情况下提供媒体查询和需要填充的图像槽宽度。然后，浏览器根据每个槽位决定加载哪些图像。例如，如果屏幕宽度小于等于 `600px`，那么 `max-width: 600px` 为真，因此需要填充的槽位是 `480px`。在这种情况下，浏览器很可能选择加载 480w.jpg 文件（480px 宽的图像）。这有助于性能提升，因为浏览器不会加载比所需更大的图像。
+
+以下示例为不同屏幕分辨率提供了不同分辨率的图像：
+
+```html
+<img
+  srcset="320w.jpg, 480w.jpg 1.5x, 640w.jpg 2x"
+  src="640w.jpg"
+  alt="家庭肖像" />
+```
+
+`1.5x`、`2x` 等是相对分辨率指示器。如果图像的样式设置为 320px 宽度（例如在 CSS 中使用 `width: 320px`），则如果设备分辨率较低（每个 CSS 像素对应一个设备像素），浏览器将加载 `320w.jpg`，如果设备分辨率较高（每个 CSS 像素对应两个或更多设备像素），则加载 `640w.jpg`。
+
+在这两种情况下，`src` 属性在浏览器不支持 `src`/`srcset` 的情况下提供了默认图像。
+
+### 为图像和视频提供不同来源
+
+{{htmlelement("picture")}} 元素基于传统的 {{htmlelement("img")}} 元素，允许你为不同的情况提供多个不同的源。例如，如果布局是宽的，你可能希望有一个宽的图像，如果是窄的，你将希望有一个在该上下文中仍然有效的较窄的图像。
+
+当然，在移动设备上，这也有助于通过提供较小的信息下载来提高性能。
+
+下面是一个示例：
+
+```html
+<picture>
+  <source media="(max-width: 799px)" srcset="narrow-banner-480w.jpg" />
+  <source media="(min-width: 800px)" srcset="wide-banner-800w.jpg" />
+  <img src="large-banner-800w.jpg" alt="茂密森林景观" />
+</picture>
+```
+
+{{htmlelement("source")}} 元素在 `media` 属性中包含媒体查询。如果媒体查询返回 true，则加载其 `<source>` 元素的 `srcset` 属性中引用的图像。在上面的示例中，如果视口宽度为 `799px` 或更小，则加载 `narrow-banner-480w.jpg` 图像。还要注意 `<picture>` 元素包含了一个 `<img>` 元素，在不支持 `<picture>` 的浏览器中提供了默认图像加载。
+
+请注意，在此示例中使用了 `srcset` 属性。如前一节所示，你可以为每个图像源提供不同的分辨率。
+
+`<video>` 元素在提供不同来源方面的工作方式类似：
+
+```html
+<video controls>
+  <source src="video/smaller.mp4" type="video/mp4" />
+  <source src="video/smaller.webm" type="video/webm" />
+  <source src="video/larger.mp4" type="video/mp4" media="(min-width: 800px)" />
+  <source
+    src="video/larger.webm"
+    type="video/webm"
+    media="(min-width: 800px)" />
+
+  <!-- 在不支持 video 元素的浏览器中的备用 -->
+  <a href="video/larger.mp4">下载视频</a>
+</video>
+```
+
+然而，提供图像和视频的源之间存在一些关键差异：
+
+- 在上面的示例中，我们使用的是 `src` 而不是 `srcset`；你无法通过 `srcset` 指定视频的不同分辨率。
+- 相反，要在不同的 `<source>` 元素中指定了不同的分辨率。
+- 请注意，我们还在不同的 `<source>` 元素中指定了不同的视频格式，每个格式都在 `type` 属性中通过其 MIME 类型进行标识。浏览器将加载它们支持（媒体查询测试返回 true）的首个格式。
+
+### 图像的懒加载
+
+提高性能的一个非常有用的技术是**懒加载**。这指的是在 HTML 渲染时不立即加载所有图像，而是仅在它们实际对用户可见（或即将可见）时加载它们。这意味着立即可见/可用的内容更快地准备就绪，而随后的内容只有在滚动到时才会渲染其图像，并且浏览器不会浪费带宽加载用户永远看不到的图像。
+
+懒加载通常使用 JavaScript 处理，但现在浏览器有一个可用的 `loading` 属性，可以指示浏览器自动进行图像的懒加载：
+
+```html
+<img src="800w.jpg" alt="家庭合照" loading="lazy" />
+```
+
+详细信息请参见 web.dev 上的[浏览器级 web 图像懒加载](https://web.dev/browser-level-image-lazy-loading/)。
```

Comment:
Could you update the url in en-US document (the previous one has been redirected)

```suggestion
详细信息请参见 web.dev 上的[浏览器级 web 图像懒加载](https://web.dev/articles/browser-level-image-lazy-loading)。
```